### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by voidove** [voidove/jug-jug#75]
## Why Accessibility?
* Accessibility is the right thing to do.

The Web represents a giant leap forward in the quality of life for people with disabilities.
For example, before the advent of online news and screen readers, it was difficult, expensive, and oftentimes, impossible, for a person with visual impairments to read a daily newspaper. That alone should be reason enough.

* Accessibility leads to good practice.

building accessible web sites has a tendency to lead towards HTML markup that is semantic and easy for everyone, even computers, to understand and interpret.

* Accessibility helps to avoid legal concerns.

Many countries have laws that require government agencies, along with public and private organizations, to maintain web content that is accessible to people of all abilities and disabilities.
